### PR TITLE
feat(file-preview-card-group): add class for card description

### PR DESCRIPTION
**Pull request by LuLaValva**
- [x] This PR contains CSS changes
- [ ] This PR does not contain CSS changes

## Description

- Add a class for the descriptions of `file-preview-card` within `file-preview-card-group`

## Screenshots

image

## Checklist

- [x] I verify the build is in a non-broken state
- [x] I verify all changes are within scope of the linked issue

- [ ] I regenerated all CSS files under dist folder
- [ ] I tested the UI in all supported browsers
- [ ] I did a visual regression check of the components impacted by doing a Percy build and approved the build
- [ ] I tested the UI in dark mode and RTL mode
- [ ] I added/updated/removed Storybook coverage as appropriate

### CODE REVIEW

File: dist/file-preview-card-group/file-preview-card-group.css
Comment (ArtBlue) [inline dist/file-preview-card-group/file-preview-card-group.css:19]: The proper BEM for this would be `file-preview-card-group__card-description`, but `file-preview-card-group` is supposed to be just a responsive layout assembler. Also, since descriptions are item-specific, this doesn't look to be the right place to add description.

I think we should add the description in `file-preview-card` as `file-preview-card__description`.

File: dist/file-preview-card-group/file-preview-card-group.css
Comment (ArtBlue) [inline dist/file-preview-card-group/file-preview-card-group.css:1]: Once the other issues are resolved, we should also add a stories in the preview card and the preview card group to cover the description.

**Review by ArtBlue — CHANGES_REQUESTED** [review]

File: dist/file-preview-card-group/file-preview-card-group.css
Comment (LuLaValva) [reply dist/file-preview-card-group/file-preview-card-group.css:19]: I agree and I wanted to do it that way, but the problem is that this is a _sibling_ of `file-preview-card` and not a child, so changing it would require breaking changes to markup.

Comment (ArtBlue) [reply dist/file-preview-card-group/file-preview-card-group.css:19]: Yikes! We should have caught this. We shouldn't have inline styles here or inside. As far as structure, I think that's the issue - it shouldn't be a _sibling_. So we don't seem to have a class used here though, which means nothing would break, right? Maybe I'm missing something.

<kbd>image</kbd>

I hesitate to keep moving in the wrong direction because we've made the wrong turn.

Comment (LuLaValva) [reply dist/file-preview-card-group/file-preview-card-group.css:19]: We don't have a class there, but I was discussing with @agliga offline about how there _should_ be since otherwise there will be a bunch of inline styles in eBayUI. I'd like for this to be called `file-preview-card__description`, but since it's a sibling that would be a violation of BEM. I don't think it makes sense to add an extra level of nesting to `file-preview-card`, but maybe you think differently?

**Comment by LuLaValva** [comment]
After some discussion offline, I ended up doing kind of a major refactor of the way that `file-preview-card` is organized. The names that changed were taken from the design side, but I'm not super happy with a few of them and would be happy to change. Please take a look.

File: dist/file-preview-card/file-preview-card.css
Comment (ArtBlue) [inline dist/file-preview-card/file-preview-card.css:78]: Was this spec-ed in `em`s? Seems like we should be using a token.

File: dist/file-preview-card/file-preview-card.css
Comment (ArtBlue) [inline dist/file-preview-card/file-preview-card.css:90]: Why switch to `em`? We should probably be using tokens, like before.

**Review by ArtBlue — COMMENTED** [review]
Looks good overall. A couple of minor comments.

The rest (naming convention) we'll discuss "offline."

File: dist/file-preview-card/file-preview-card.css
Comment (LuLaValva) [reply dist/file-preview-card/file-preview-card.css:78]: It really just needs to be any number larger than the height so edges will be rounded, I used `em` since that will work even if a chrome extension or something changes font size to be larger.

File: dist/file-preview-card/file-preview-card.css
Comment (LuLaValva) [reply dist/file-preview-card/file-preview-card.css:90]: Yeah, not sure why I changed this. I'll switch back to a token.

File: src/modules/file-preview-card.marko
Comment (agliga) [inline src/modules/file-preview-card.marko:11]: Is this width needed? I know it was here before but shouldn't we have sizes based on the grid for this?

File: dist/file-preview-card/file-preview-card.css
Comment (agliga) [inline dist/file-preview-card/file-preview-card.css:18]: Are we going to stick with grid or go to using position absolute?

Comment (LuLaValva) [reply dist/file-preview-card/file-preview-card.css:18]: I just switched to absolute positioning, I'll change some names and then we should be ready for review

File: src/modules/file-preview-card.marko
Comment (LuLaValva) [reply src/modules/file-preview-card.marko:11]: This is an arbitrary width just for the example. The container of the component is what decides its width

**Review by agliga — APPROVED** [review]
LGTM, good job on the changes!

**Review by ArtBlue — APPROVED** [review]
Looks great! Thanks for bearing with us. 😃 

Just add a changeset and check all the items off from the dev checklist.

Let's run a targeted Percy build too since this was out already to ensure visual consistency.

**Comment by agliga** [comment]
> Looks great! Thanks for bearing with us. 😃 
> 
> 
> 
> Just add a changeset and check all the items off from the dev checklist.
> 
> 
> 
> Let's run a targeted Percy build too since this was out already to ensure visual consistency. 

I give my approval as long as the percy tests pass. Feel free to merge this once Arthur approves again. 
Good work!

**Review by ArtBlue — APPROVED** [review]
Looks good!

**Pull request merged**
